### ISSUE TITLE
[Button]: accessibility issues related to the component

### ISSUE THREAD

**Issue by NatalliaAlieva** [epam/UUI#1492]
### Description
Icon-button [View] has no accessible name.

### Precondition
Screen Reader is **ON**

### Steps to Reproduce
1. Open [Button](https://uui.epam.com/documents?id=button&mode=doc&skin=UUI3_loveship&category=components) component.
2. Use the Tab key to navigate to Icon-buttons [View] and note the announcement. 

### Actual Result
Icon-buttons [View] are announced without accessible name just as "button".
<img width="1177" alt="Screenshot 2023-07-14 at 13 11 53" src="https://github.com/epam/UUI/assets/57751283/104453f8-bf32-49e3-8301-a5064e805bcc">

### Expected Result
Icon-buttons [View] should be announced with accessible name that clearly describe its purpose, e.g., View button or View <purpose> button. 

**Failed WCAG Checkpoint:** 4.1.2 Name, Role, Value (Level A)
https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html
